compiler/z/codegen/OMRTreeEvaluator.hpp

```diff
@@ -254,6 +254,41 @@ class OMR_EXTENSIBLE TreeEvaluator: public OMR::TreeEvaluator
    static TR::Register *MethodExitHookEvaluator(TR::Node *node, TR::CodeGenerator *cg);
    static TR::Register *PassThroughEvaluator(TR::Node *node, TR::CodeGenerator *cg);
 
+   /** \brief
+    *    This is a helper function used for floating point max/min operations
+    *    when SIMD instructions are available. +0.0 compares as strictly
```

Comment:
I think we should mention that it generates Vector Instructions when available for max/min operations.